src/lib/model/connections.js

```diff
@@ -26,14 +26,22 @@ import bondingModeEnum from '../model/bondingMode';
 
 let connectionIndex = 0;
 
+/**
+ * @typedef {Object} IPConf
+ * @property {Array<Object>} addresses - Address configurations
+ * @property {string} bootProto - Boot protocol (@see model/bootProtocol)
+ */
+
 /**
  * @typedef {Object} Connection
  * @property {string} name - Connection name
  * @property {string} description - Connection description
  * @property {string} type - Connection type (@see model/interfaceType)
- * @property {string} bootProto - Boot protocol (@see model/bootProtocol)
  * @property {string} interfaceName - Associated interface name
- * @property {Array<Object>} addresses - Address configurations
+ * @property {string} startMode - the connection start mode (@see model/startMode)
+ * @property {IPconf} ipv4 - IPv4 configuration
+ * @property {IPconf} ipv6 - IPv6 configuration
+ * @property {boolean} exists - Whether the connection was loaded from the system or not
```

Comment:
@imobachgs, @teclator, 

What about using _fake_, _mock_ , _isMock_, _simulated_, or [whatever other option](https://www.thesaurus.com/browse/fake?s=t) instead of _exists_?

```js

if (connection.isMock) {
  doSomething();
}
```